src/dropbox-base.js

```diff
@@ -3,6 +3,9 @@ import { downloadRequest } from './download-request';
 import { uploadRequest } from './upload-request';
 import { rpcRequest } from './rpc-request';
 
+// Expiration is 300 seconds but needs to be in milliseconds for Date object
+const TokenExpirationBuffer = 300 * 10000;
```

Comment:
Shouldn't this be `300 * 1000`?

Reply:
nit: Also I think just `TokenExpiration` is a better name.

Reply:
This is meant to be a buffer, we attempt to refresh if the token is within this buffer rather than just if it is already expired

Reply:
Oh got it. Makes sense.